src/controllers/controllerengine.cpp

```diff
@@ -1440,6 +1440,22 @@ void ControllerEngine::softTakeover(QString group, QString name, bool set) {
 }
 
 /*  -------- ------------------------------------------------------
+     Purpose: Ignores the next value for the given ControlObject
+                This is used when an absolute physical control is changed to
+                to operate on a different ControlObject
+     Input:   ControlObject group and key values
+     Output:  -
+     -------- ------------------------------------------------------ */
+void ControllerEngine::softTakeoverIgnoreNext(QString group, QString name) {
```

Comment:
softTakeoverIgnoreNext() might be a puzzel if you do not know the Implementation details. 

This function must we called whenever a absolute control takes over a ControlObject which was controlled from elsewhere and which is likely out of sync because of that. 
It would be nice if we can relfect something of this in the function name.

softTakeoverRequireResync()
softTakeoverDetach() 
softTakoverLoose()  
softTakoverAtacheOtherControl()